packages/@react-aria/select/src/useSelect.ts

```diff
@@ -50,9 +59,11 @@ interface SelectAria {
  * @param props - Props for the select.
  * @param state - State for the select, as returned by `useListState`.
  */
-export function useSelect<T>(props: AriaSelectOptions<T>, state: SelectState<T>, ref: RefObject<HTMLElement>): SelectAria {
```

Comment:
Yep, as you noted in https://github.com/adobe/react-spectrum/pull/1250#issuecomment-724986231 this would be a breaking change, so we'll need to keep this ref and maybe add the new on in the props. I see you got it from useComboBox as well.
Maybe we can support both signatures and mark the current one if used as Deprecated
@dannify thoughts?

Reply:
I've restore the original API and added the `popoverRef` as optional, so the API contract is not broken any more